README

```diff
@@ -39,23 +39,25 @@ conforming and non-conforming (AMR) adaptive refinement. Arbitrary element
 transformations, allowing for high-order mesh elements with curved boundaries,
 are also supported.
 
-MFEM is commonly used as a "finite element to linear algebra translator", since
-it can take a problem described in terms of finite element-type objects, and
-produce the corresponding linear algebra vectors and sparse matrices. In order
-to facilitate this, MFEM uses compressed sparse row (CSR) sparse matrix storage
-and includes simple smoothers and Krylov solvers, such as PCG, MINRES and GMRES,
-as well as support for sequential sparse direct solvers from the SuiteSparse
-library. Nonlinear solvers (the Newton method), eigensolvers (LOBPCG), and
-several explicit and implicit Runge-Kutta time integrators are also available.
+When used as a "finite element to linear algebra translator", MFEM can take a
+problem described in terms of finite element-type objects, and produce the
+corresponding linear algebra vectors and fully or partially assembled operators,
+e.g. in the form of global sparse matrices or matrix-free operator evaluation.
+The library includes simple smoothers and Krylov solvers, such as PCG, MINRES
+and GMRES, as well as support for sequential sparse direct solvers from the
+SuiteSparse library. Nonlinear solvers (the Newton method), eigensolvers
+(LOBPCG), and several explicit and implicit Runge-Kutta time integrators are
+also available.
 
 MFEM supports MPI-based parallelism throughout the library, and can readily be
-used as a scalable unstructured finite element problem generator. MFEM-based
-applications require minimal changes to transition from a serial to a
-high-performing parallel version of the code, where they can take advantage of
-the integrated scalable linear solvers from the hypre library. Comprehensive
-support for other external packages, e.g. PETSc and SUNDIALS is also included,
-giving access to many additional linear and nonlinear solvers, preconditioners,
-time integrators, etc.
+used as a scalable unstructured finite element problem generator. As of version
+4.0, MFEM also supports GPU acceleration, and programming models, such as CUDA,
```

Comment:
I think it is more accurate to say that "... MFEM offers an initial, limited support for GPU ..."; otherwise, we are stretching the meaning of the word "supports".

Reply:
" initial, limited support" seems to strong to me, I just said "initial support"